app/controllers/api/v2/users_controller.rb

```diff
@@ -111,6 +111,11 @@ def destroy
         end
       end
 
+      api :GET, "/users/menu", N_("Get menu items for current user")
```

Comment:
Won't this overlap with a username `menu`? Or can you only retrieve users by ID.

Reply:
Good catch, I changed it to `/menu`.